src/main/java/com/depromeet/domain/missionRecord/application/MissionRecordService.java

```diff
@@ -83,7 +84,11 @@ public MissionRecordFindOneResponse findOneMissionRecord(Long recordId) {
                 missionRecordRepository
                         .findById(recordId)
                         .orElseThrow(() -> new CustomException(ErrorCode.MISSION_RECORD_NOT_FOUND));
-        return MissionRecordFindOneResponse.from(missionRecord);
+        long sinceDay =
+                Duration.between(missionRecord.getMission().getStartedAt(), LocalDateTime.now())
+                                .toDays()
+                        + 1;
```

Comment:
상단에 매직넘버 상수로 빼는게 좋을 것 같아용
`private static final int DAYS_ADJUSTMENT = 1;`

```suggestion
        long sinceDay =
                Duration.between(missionRecord.getMission().getStartedAt(), LocalDateTime.now())
                                .toDays() + DAYS_ADJUSTMENT;
```

또는

```suggestion
        long sinceDay =
                Duration.between(missionRecord.getMission().getStartedAt(), LocalDateTime.now().minusDays(DAYS_ADJUSTMENT))
                                .toDays();
```
와 같이 처리할수도 있을 것 같네용

Reply:
1번 내용으로 수정하겠습니당